### PR TITLE
CT-1552 Update Business Units that can handle SAR cases

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: lib/tasks/kilo_map.rake [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  i = 0
+  CSV.foreach(File.join(Rails.root, 'lib', 'assets', 'kilo_map.csv')) do |row|
+    puts row.inspect
+    next if header_row?(row)
+    bg, bu, sar = row
+    if bg.present?
+     @bg = BusinessGroup.find_by_name bg
+     if @bg.nil?
+       puts ">>>>>>>>>>>> UNABLE TO FIND BUSINESS GROUP #{bg} #{__FILE__}:#{__LINE__} <<<<<<<<<<<<\n"
+    else
```
Comment (misaka) [inline lib/tasks/kilo_map.rake]: This doesn't look correctly indented.

File: lib/assets/kilo_map.csv [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+,Contract & Performance Team,YES
+,Crime Directorate,YES
+,Customer Investigations,YES
+,Adjudicator to HM Land Registry,YES
+,Asylum Support Team,YES
+,DO NOT USE - TO BE DELETED,
+,First Tier Tribunal - Asylum & Immigration Tribunal (IAC),YES
+,First Tier Tribunal - Mental Health Tribunal,YES
+,First-tier Tribunal - Property Chamber - Residential Property (FtT (PC) RP),YES
+,"First Tier Tribunal - Special Tribunals (Gender,
```
Comment (misaka) [inline lib/assets/kilo_map.csv]: Is there a newline in this business unit name??

File: lib/assets/kilo_map.csv [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Information, Gambling, Charity)",YES
+,First Tier Tribunal - Tax Chamber,YES
+,Social Security & Child Support Tribunal (SSCS),YES
+,"Special Education Needs, Care Standard & Primary, Health Lists Tribunal (SENDIST)",YESq
```
Comment (misaka) [inline lib/assets/kilo_map.csv]: Extra `q` at the end here.

**Comment by misaka** [comment]
Just realised this is WIP. Might be still worth looking at the issues pointed-to above.

**Review by beccapearce — APPROVED** [review]

**Review by misaka — APPROVED** [review]

**Pull request merged**